.rubocop_todo.yml

```diff
@@ -23,7 +23,7 @@ Metrics/BlockLength:
 # Offense count: 1
 # Configuration parameters: CountComments.
 Metrics/ClassLength:
-  Max: 284
+  Max: 285
```

Comment:
wonder if we should add an exclude for catalog_controller.rb here too

Reply:
yeah, makes sense, will do